shell/src/main/java/alluxio/cli/fs/command/LsCommand.java

```diff
@@ -317,7 +318,7 @@ public int run(CommandLine cl) throws AlluxioException, IOException {
 
   @Override
   public String getUsage() {
-    return "ls [-d|-f|-p|-R|-h|--sort=option|--timestamp=option|-r] <path> ...";
+    return "ls [-d|-f|-p|-R/--recursive|-h|--sort=option|--timestamp=option|-r] <path> ...";
```

Comment:
can we all use | instead of / as the separator?

Reply:
On both sides of “｜”, there are options for different functions. For example, -d and -f have different meanings, but -R and --recursive are essentially the same options, so I think it might be better to use "/".

Reply:
it's more a coding style issue. In this case -r is also there using |, isn't them used for the same recursive meaning?

Reply:
Here, -r means reverse, not recursive.

Reply:
Maybe we can use other ways to distinguish, such as adding (), what do you think?